tests/home/home.spec.ts

```diff
@@ -10,46 +10,46 @@ test('Home page displays all of its elements', async ({ homePage }) => {
 
 test('Do simple metadata search', async ({ homePage }) => {
   await test.step(`Query for "cats" and validate that "cats" appears as the search term `, async () => {
-    await homePage.searchPage.queryFor('cats');
-    await homePage.searchPage.validateSearchInput('cats');
+    await homePage.collectionBrowser.queryFor('cats');
+    await homePage.collectionBrowser.validateSearchInput('cats');
```

Comment:
Q: What is `homePage.collectionBrowser`? To my knowledge there shouldn't be an instance of `<collection-browser>` present on the home page.

Reply:
thanks for pointing that out.
this should be a reused page object for `collection-search-input` which was defined in `collection-browser` page object

Reply:
i'll do the refactor for this in another PR instead